src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorIT.java

```diff
@@ -193,24 +192,11 @@ private void validateIndexIngestResults(String indexName, String fieldName, Obje
         assertEquals(expected, ingestOutputs);
     }
 
-    private void createPipelineProcessor(String pipelineName) throws Exception {
+    private void createPipelineWithChunkingProcessor(String pipelineName) throws Exception {
```

Comment:
Can we shift this method in BaseNeuralSearchIT? and reuse the already existing methods createPipelineProcessor by passing modelId as an argument?

Reply:
I have update the function to call existing method like `createPipelineProcessor(requestBody, pipelineName, "");`. I do not think there is much need to move this method to BaseNeuralSearchIT because currently there are no more tests using this method. If BWC tests need to create a chunking processor, they have their own method.